locations/spiders/coffee_1_gb.py

```diff
@@ -0,0 +1,68 @@
+import re
+
+import scrapy
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours, day_range
+
+
+class Coffee1GB(scrapy.Spider):
+    name = "coffee_1_gb"
+    item_attributes = {"brand": "Coffee#1", "brand_wikidata": "Q22032058"}
+    allowed_domains = ["www.coffee1.co.uk"]
+    start_urls = ["https://www.coffee1.co.uk/locations/get"]
+
+    def parse(self, response):
+        for item in response.json():
+            if " - closed" in item["post_title"].lower():
+                continue
+            yield scrapy.Request(item["url"], callback=self.parse_branch, meta={"item": item})
+
+    def parse_branch(self, response):
+        location = response.meta["item"]
+        item = DictParser.parse(location)
+
+        if "<br />" in location["branch_address"]:
+            address = [i.strip(" ,") for i in location["branch_address"].split("<br />\r\n") if i]
+        else:
+            address = location["branch_address"].split(", ")
+
+        if "branch-fallback" not in location["branch_image"]:
+            item["image"] = location["branch_image"]
+
+        item["ref"] = re.search(r"/locations/([^/]+)/$", location["url"]).group(1)
+        item["name"] = location["post_title"]
+        item["city"] = location["branch_city"]
+        item["addr_full"] = "\n".join(address)
```

Comment:
We typically split addr_full with commas.

```suggestion
        item["addr_full"] = ", ".join(address)
```